daffodil-test/src/test/resources/org/apache/daffodil/section00/general/testTextBidi.tdml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<tdml:testSuite xmlns:tdml="http://www.ibm.com/xmlns/dfdl/testData"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ex="http://example.com"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:daf="urn:ogf:dfdl:2013:imp:daffodil.apache.org:2018:ext"
+  suiteName="unparser tests involving bitOrder and outputValueCalc interactions"
+  defaultRoundTrip="false">
```

Comment:
Can you make this onePass so it also tests the unparse side?

Reply:
Good catch @olabusayoT. This is a valuable change to make to ensure all tests work for both parse and unparse.

Also since you're already making changes to this file, I'd also suggest combining the defineSchemas into one. In this case, it shoudn't be necessary to have two different schemas. Instead, just have a single defineSchema with two root elements (suggest changing their names from ``test`` and ``root`` to better differentiate) and then the tdml:parserTestCase's can reference the same model with a different root element.

Reply:
Good comments. Done.